### PR TITLE
fixed message order

### PR DESCRIPTION
**Pull request by akinolu52**
this Pr will fix the ordering of messages in a chat view

### CODE REVIEW

File: server/routes/api/conversations.js [hunk 1]
```diff
@@ -73,10 +73,10 @@ router.get("/", async (req, res, next) => {
...
     }
 
-    // sort the messages 
-    conversations?.messages?.sort((a, b) => {
-      return new Date(a?.createdAt) - new Date(b?.createdAt);
-    });
+    // // sort the messages 
+    // conversations?.messages?.sort((a, b) => {
+    //   return new Date(a?.createdAt) - new Date(b?.createdAt);
+    // });
```
Comment (FilipeGorodscy) [inline server/routes/api/conversations.js]: You should remove commented-out code from your PRs.

File: client/src/store/utils/thunkCreators.js [hunk 2]
```diff
@@ -72,6 +72,10 @@ export const logout = (id) => async (dispatch) => {
 export const fetchConversations = () => async (dispatch) => {
   try {
     const { data } = await axios.get("/api/conversations");
+    for (let index = 0; index < data.length; index++) {
+      const { messages } = data[index];
+      messages?.sort((a, b) => new Date(a?.createdAt) - new Date(b?.createdAt));
+    }
```
Comment (FilipeGorodscy) [inline client/src/store/utils/thunkCreators.js]: Could you rewrite this loop using `forEach`?

**Review by FilipeGorodscy — APPROVED** [review]
I'll approve this PR since you've already completed the sending message bug [here](https://github.com/akinolu52/turbo-rotary/pull/5). Please see the comments below and feel free to merge after addressing them :)

File: client/src/store/utils/thunkCreators.js (hunk 2 above)
Comment (akinolu52) [reply client/src/store/utils/thunkCreators.js]: okay sure, I'll

File: server/routes/api/conversations.js (hunk 1 above)
Comment (akinolu52) [reply server/routes/api/conversations.js]: okay, thanks

**Pull request merged**